cfme/common/vm.py

```diff
@@ -525,6 +526,17 @@ def unset_ownership(self):
             action=form_buttons.save)
         flash.assert_success_message('Ownership saved for selected {}'.format(self.VM_TYPE))
 
+    def open_console(self, console='Web Console', invokes_alert=False, cancel=False):
```

Comment:
You're absolutely correct - console interaction doesn't work, this just opens the console. Still working on console view/control, wanted to merge this in so it didn't sit around and get stale. Thoughts? Maybe not worth merging separately?